docs/src/main/asciidoc/_project-features-wiremock.adoc

```diff
@@ -63,6 +63,12 @@ NOTE: Actually, WireMock always loads mappings from `src/test/resources/mappings
 well as* the custom locations in the `stubs` attribute. To change this behavior, you can
 also specify a files root, as described in the next section of this document.
 
+NOTE: Also, the mappings in the `stubs` location are not considered part of Wiremock's "default mappings" and calls
```

Comment:
This seemed to be the "best" location to add the info. I was going to add to the annotation but it seemed a little bit "specific" for a unusual use case to put in there.